pesu-components/src/components/Modal/Modal.styles.tsx

```
@@ -0,0 +1,88 @@
+import styled from '@emotion/styled';
+import type { ModalInterface } from './Modal';
+
+const getModalWidth = (size: 'small' | 'medium' | 'large', position: 'center' | 'bottom') => {
+  if (position === 'center') {
+    switch (size) {
+      case 'small':
+        return '320px';
+      case 'large':
+        return '600px';
+      case 'medium':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)